01-shapefile-attributes.Rmd

```diff
@@ -1,15 +1,18 @@
 ---
 layout: post
-title: "Lesson 01: Work With Shapefile Attributes in R"
+title: "Lesson 01: Shapefile Metadata & Attributes in R"
 date:   2015-10-25
 authors: [Joseph Stachelek, Leah Wasser]
 contributors: [Megan A. Jones, Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
+packagesLibraries: [rgdal, raster]
 category: 
 tags: [module-1]
 mainTag: GIS-Spatial-Data
-description: "This lesson explains the nature of shapefile attributes. Participants will be able to locate and query shapefile attributes as well as subset shapefiles by specific attribute values."
+description: "This lesson explains the nature of shapefile attributes. The
```

Comment:
This lesson overviews how to locate and query shapefile attributes as well as subset shapefiles by specific attribute values in R.